frontend/main-app/src/components/DashboardAccordion/DashboardAccordion.tsx

```diff
@@ -95,7 +56,7 @@ export const DashboardAccordion: React.FC<DashboardAccordionProps> = ({ form })
             <CustomParagraph heading="Expiration date" content={form.expires || "Not set"} />
             <CustomParagraph heading="Submissions" content={submissionCount} />
           </div>
-          <ButtonGroup submissions={submissions} formTitle={form.title} formUrl={formUrl} />
+          <ButtonGroup submissions={submissions} form={form} formUrl={formUrl} />
```

Comment:
It looks like only the title is used, is there a reason for this change?

Reply:
In PR #146, I'm adding a new context, `FormContext`, that removes much of the prop drilling in the dashboard page.  In that PR, many a `formTitle` was removed. Guess I got carried away 😅